src/UserService.Models.Dto/UserRequest.cs

```diff
@@ -14,6 +16,7 @@ public class UserRequest
         public string Password { get; set; }
         public bool IsAdmin { get; set; }
         public bool IsActive { get; set; }
+        public List<UserConnection> Connections { get; set; }
```

Comment:
maybe it's better to use IEnumerable instead of List here

Reply:
yes it's possible, but it seemed easier for me using the list(it also works)

Reply:
this will also work with IEnumerable because you don't use methods that aren't present in this interface. I think it is not critical to use List, but just in other classes was used IEnumerable

Reply:
let's ask @djino97 about this